ext/mkrf_conf.rb

```diff
@@ -0,0 +1,21 @@
+require "rubygems/dependency_installer"
+
+di = Gem::DependencyInstaller.new
+
+begin
+  if RUBY_PLATFORM == "java"
+    puts "Installing curses because Ruby #{RUBY_VERSION}"
+    di.install "activerecord-jdbcsqlite3-adapter"
+  end
+rescue => e
+  warn "#{$0}: #{e}"
+  
+  exit!
+end
+
+puts "Writing fake Rakefile"
+
+# Write fake Rakefile for rake since Makefile isn't used
+File.open(File.join(File.dirname(__FILE__), 'Rakefile'), 'w') do |f|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.